data/frame.go

```diff
@@ -202,13 +202,21 @@ func (f *Frame) CopyAt(fieldIdx int, rowIdx int) interface{} {
 	return f.Fields[fieldIdx].vector.CopyAt(rowIdx)
 }
 
-// Set set the val to the specified fieldIdx and rowIdx.
+// Set sets the val to the specified fieldIdx and rowIdx.
 // It will panic if either the fieldIdx or rowIdx are out of range or
 // if the underlying type of val does not match the element type of the Field.
 func (f *Frame) Set(fieldIdx int, rowIdx int, val interface{}) {
 	f.Fields[fieldIdx].vector.Set(rowIdx, val)
 }
 
+// SetConcreateAt sets the val to the specified fieldIdx and rowIdx.
```

Comment:
If we like comment change for Field, should update this to match. Also this method needs spelling fix as well.